commands/list.go

```diff
@@ -0,0 +1,115 @@
+package commands
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/platformsh/cli/internal/legacy"
+)
+
+func init() {
+	RootCmd.AddCommand(ListCmd)
+}
+
+func init() {
+	ListCmd.Flags().String("format", "txt", "The output format (txt, json, or md) [default: \"txt\"]")
+	ListCmd.Flags().Bool("raw", false, "To output raw command list")
+	ListCmd.Flags().Bool("all", false, "Show all commands, including hidden ones")
+
+	viper.BindPFlags(ListCmd.Flags()) //nolint:errcheck
+}
+
+const projectNamespace = "project"
+
+var ListCmd = &cobra.Command{
+	Use:   "list [flags] [namespace]",
+	Short: "List",
```

Comment:
```suggestion
	Short: "Lists commands",
```

This is the current description

Reply:
fixed